### ISSUE TITLE
delete don't clear letters on Tileset

### ISSUE THREAD

**Issue by alexgoldblat** [ricardoquesada/vchar64#21]
could be useful and handy if i could clear letters with "Delete" key

**Comment by ricardoquesada** [comment]
you to clear the entire tile, similar to Shift + Command + C ?

**Comment by alexgoldblat** [comment]
Ricardo , i prefer Delete will clear the selected tile if possible

**Comment by ricardoquesada** [comment]
I just wanted to make sure that you are talking about that functionality.

**Comment by alexgoldblat** [comment]
yes. for i don't want to Cut and than paste. only delete!

**Comment by ricardoquesada** [comment]
Shift + Command + C is only delete. have you tried? in which way you like it different?
Please, detail the use case. thanks.

**Comment by alexgoldblat** [comment]
Hi Ricardo,
yes it work with SHIFT+CTRL+C ! . but when i choose more than 1 selection
so it only delete the first tile.
btw Command is not CTRL as far as i know but okay :)  thats why i didn't
understand why you keep writing Command.
normally command are from Laptops so they have a special key which enable
functions . never mind :)
if you can make "Delete" as like SHIFT+CTRL+C could be nice :)

**Comment by ricardoquesada** [comment]
yes, "clear" affects only one tile. All the Tile commands affect just one tile, not the selected region.

You can see all the available Tile commands and its keyboard shortcuts here:

VChar -> Tile

or in its documentation:

https://github.com/ricardoquesada/vchar64/wiki/Documentation#tile-manipulation

**Comment by alexgoldblat** [comment]
Hi Ricardo !
Ok thanks for the keyboard help. cool . you want me to close the ticket for
the Delete ?

**Comment by ricardoquesada** [comment]
it could be renamed to: "add 'delete' as a shortcut to Tile Clear as well"

**Comment by alexgoldblat** [comment]
Ok. done !
